src/components/search-query.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import algoliasearch from 'algoliasearch';
+import PropTypes from 'prop-types';
+
+import { RefinementList, InstantSearch, SearchBox, Hits, ClearRefinements, Pagination } from 'react-instantsearch-dom';
+import webComponent from '../web-component';
+
+const searchClient = algoliasearch('H897LKXYG1', '50a4f124e0d934cac92e79ece376316a');
+
+const hitComponent = ({ hit }) => (
+  <div>
+    { hit.last_name ?
+      <div><strong>Contributor:</strong> {hit.first_name} {hit.last_name}</div> : ''
+    }
+    <div><strong>Election: </strong>
+      <a href={`/election/${hit.election_location.toLowerCase()}/${hit.election_date}`} >
+        {hit.election_title}
+      </a>
+    </div>
+    { hit.title ?
+      <div><strong>Ballot Measure: </strong>
+        <a href={`/referendum/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.slug}`} >
+          {hit.title}
+        </a>
+      </div> : ''}
+    { hit.name ?
+      <div><strong>Candidate: </strong>
+        <a href={`/candidate/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.candidate_slug}`} >
+          {hit.name}
+        </a>
+      </div>
+     : ''
+    }
+    { hit.office_title ?
+      <div>
+        <strong>Office: </strong>
+        <a href={`/office/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.office_slug}`} >
+          {hit.office_title}
+        </a>
+      </div> : ''
+    }
+    { hit.amount ? <div><strong>Amount:</strong> ${hit.amount}</div> : ''}
+    <strong>Election date:</strong> {hit.election_date}<br />
+    <strong>Location:</strong> {hit.election_location}<br />
+  </div>
+);
+hitComponent.propTypes = {
+  hit: PropTypes.shape({
+    last_name: PropTypes.string,
+    first_name: PropTypes.string,
+    election_title: PropTypes.string,
+    election_location: PropTypes.string,
+    election_date: PropTypes.string,
+    name: PropTypes.string,
+    title: PropTypes.string,
+    amount: PropTypes.number,
+    slug: PropTypes.string,
+    candidate_slug: PropTypes.string,
+    office_slug: PropTypes.string,
+    office_title: PropTypes.string,
+  }),
+};
+hitComponent.defaultProps = {
+  hit: {},
+};
+
+class SearchQuery extends React.Component {
+  render() {
+    return (
+      <InstantSearch searchClient={searchClient} indexName="election">
+        <div className="left-panel">
+          <ClearRefinements clearsQuery="true" />
+          <h3> Election Title </h3>
+          <RefinementList attribute="election_title" />
+        </div>
+        <div className="right-panel">
+          <SearchBox className="searchbar" />
+          <Hits hitComponent={hitComponent} />
+          <Pagination />
+          <a href="https://www.algolia.com" >Search by Algolia</a>
+        </div>
+      </InstantSearch>
+    );
+  }
+}
```

Comment:
To resolve the eslint error that's causing the CircleCI build to fail, we can simply rewrite this component to be a function (stateless) component instead of a class component. This [SO answer](https://stackoverflow.com/a/43379215) has a good explanation of the error and ways to resolve it. Here's an implementation of the function component: 

```suggestion
const SearchQuery = () => (
  <InstantSearch searchClient={searchClient} indexName="election">
    <div className="grid">
      <div className="grid-col-2 election-checkboxes">
        <ClearRefinements clearsQuery="true" />
        <h3>Election Title</h3>
        <RefinementList attribute="election_title" />
      </div>
      <div className="grid-col-10">
        <SearchBox className="searchbar" />
        <Hits hitComponent={HitComponent} />
        <Pagination />
        <a href="https://www.algolia.com">Search by Algolia</a>
      </div>
    </div>
  </InstantSearch>
);
```

**Note:** This updated component also has the styling updates that I recommended above. See the classNames that now include `grid`, `grid-col-2`, `grid-col-10`, and `election-checkboxes`. I ran this code and it looks nearly identical to the styles that were implemented prior, with the added benefit of side margins so the content isn't pressed against the edges of the screen! 

<img width="1440" alt="Screen Shot 2020-07-26 at 12 27 07 PM" src="https://user-images.githubusercontent.com/28023047/88487580-5f78e500-cf3b-11ea-83a6-162499fe4c24.png">
<img width="400" alt="Screen Shot 2020-07-26 at 12 28 28 PM" src="https://user-images.githubusercontent.com/28023047/88487602-87684880-cf3b-11ea-9ae8-17b70c82fc46.png">